### PR TITLE
fix: fix the renaming error in squeeze converter

### PR DESCRIPTION
**Pull request by bowang007**
Signed-off-by: Bo Wang <bowa@nvidia.com>

# Description
Some converters may have the risks of renaming ITensors, which might cause errors. 

Fixes: #930 , #982

More PRs related to this PR: 
#1004 , #1167 ,  

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: core/conversion/converters/impl/squeeze.cpp [hunk 1]
```diff
@@ -26,7 +26,8 @@ auto squeeze_registrations TORCHTRT_UNUSED = RegisterNodeConversionPatterns().pa
        }
 
        if (selfDim[dim] != 1) {
-         auto out = ctx->AssociateValueAndTensor(n->outputs()[0], self);
+         auto id_layer = ctx->net->addIdentity(*self);
```
Comment (peri044) [inline core/conversion/converters/impl/squeeze.cpp:29]: You could use already existing utility for this https://github.com/pytorch/TensorRT/blob/d3041f553b60cf8639117e77cb9a68675755c307/core/conversion/converters/converter_util.cpp#L124

Comment (bowang007) [reply core/conversion/converters/impl/squeeze.cpp:29]: thanks @peri044. BTW, do we still need this test case https://github.com/pytorch/TensorRT/blob/078977c5a24b5149bfb5aa8996982a5b072f4220/tests/core/conversion/converters/test_squeeze.cpp#L29? I'm thinking of deleting it.

**Comment by ncomly-nvidia** [comment]
@narendasan @bowang007 is it possible to get this into v1.2? Would be good to close out this issue in this release.

File: core/conversion/converters/impl/squeeze.cpp (hunk 1 above)
Comment (bowang007) [reply core/conversion/converters/impl/squeeze.cpp:29]: @peri044  it seems that we should have an explicit name for output tensor if we want to use this util: https://github.com/pytorch/TensorRT/blob/d3041f553b60cf8639117e77cb9a68675755c307/core/conversion/converters/converter_util.cpp#L124. 
I checked around but have no idea what name would be appropriate, any idea?

**Review by narendasan — APPROVED** [review]
This seems fine, are there any other binding name changes that need to be merged?

**Comment by bowang007** [comment]
> This seems fine, are there any other binding name changes that need to be merged?

I don't think so.

**Comment by peri044** [comment]
There are merge conflicts and these changes are already merged via another PR https://github.com/pytorch/TensorRT/pull/1167

**Pull request closed without merging**